docs/usage/Analytics.md

```diff
@@ -0,0 +1,83 @@
+# Analytics
+Starting with CAM 1.3.0 it is possible to run an analysis that will show the number of Kubernetes resources, images, and pv capacity. A break down will be provided per namespace, with a break down of individual kubernetes resources in each namespace.
```

Comment:
s/CAM/MTC

at some point we'll likely need to do a broader search/replace for CAM to MTC in our docs. 
Might as well get in the habit of saying MTC now :)